src/components/DetailsSection/style.css

```
@@ -0,0 +1,6 @@
+.details-container {
+padding-left: 350px;
+padding-right: 300px;
+padding-top: 50px;
+}
```

Identar bien el CSS.

```suggestion
.details-container {
  padding-left: 350px;
  padding-right: 300px;
  padding-top: 50px;
}
```